src/cookies.js

```diff
@@ -268,10 +268,24 @@ function checkOriginForSettingCookie(win, options, name) {
  */
 function getTempCookieName(win) {
   let testCookieName = TEST_COOKIE_NAME;
-  const counter = 0;
+  let counter = 0;
   while (getCookie(win, testCookieName)) {
     // test cookie name conflict, append counter to test cookie name
-    testCookieName = TEST_COOKIE_NAME + counter;
+    testCookieName = TEST_COOKIE_NAME + counter++;
   }
   return testCookieName;
 }
+
+/**
+ * @param {!Window} win
+ * @return {boolean}
+ */
+export function canSetCookie(win) {
```

Comment:
non blocking, but can this value be memoized in some fashion?
would be happy with a comment/note if yes.

Reply:
I'm sure it's a tiny edge case, but strictly speaking, a user can disable cookies on their browser at any time, invalidating our previously memoized value.

Reply:
makes sense.